wire-ios-data-model/Source/Model/Message/ZMMessage+Internal.h

```diff
@@ -32,6 +32,13 @@
 @class ZMReaction;
 @class ZMClientMessage;
 
+typedef NS_ENUM(NSInteger, ZMExpirationReason) {
+    ZMExpirationReasonOther = 0,
```

Comment:
`other` replaces `unknown` as usually the reason for expiring is known but we don't need so much granularity.